src/App.js

```diff
@@ -101,6 +111,126 @@ function App() {
                     </RequireAuth>
                   }
                 />
+                <Route
+                  path="/monitoring"
+                  element={
+                    <RequireAuth>
+                      <Overview theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/overview"
+                  element={
+                    <RequireAuth>
+                      <Overview theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/uci-api"
+                  element={
+                    <RequireAuth>
+                      <Uciapi theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/inbound"
+                  element={
+                    <RequireAuth>
+                      <Inbound theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/orchestrator"
+                  element={
+                    <RequireAuth>
+                      <Orchestrator theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/broadcast-transformer"
+                  element={
+                    <RequireAuth>
+                      <BroadcastTransformer theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/outbound"
+                  element={
+                    <RequireAuth>
+                      <Outbound theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/transformer"
+                  element={
+                    <RequireAuth>
+                      <Transformer theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/logs"
+                  element={
+                    <RequireAuth>
+                      <UCIAPIlogs theme={store?.theme} />
+                    </RequireAuth>
+                  }
+                />
+                <Route
+                  path="/monitoring/logs/uci-api"
+                  element={
```

Comment:
why same component have 2 different routes ?

Reply:
Monitoring is the main section and in that we have subsection. so if someone clicks on monitoring, if we do not have any route for it, it will show an err. So, just for that added the route of the first subsection